src/app/plugins/scores.plugin.ts

```diff
@@ -0,0 +1,180 @@
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+
+export class ScoresPlugin extends Plugin {
+  public name: string = 'NCAA Scores Plugin';
+  public description: string = 'Gets score of a football game.';
+  public usage: string = 'scores <sport> <team name>; ex scores NCAA UCF';
+  public permission: ChannelType = ChannelType.Public;
+  public pluginChannelName: string = Constants.Channels.Public.Sports;
+
+  private _NCAA_LINK: string = 'http://www.espn.com/college-football/bottomline/scores';
+  private _NFL_LINK: string = 'http://www.espn.com/nfl/bottomline/scores';
+  private _MLB_LINK: string = 'http://www.espn.com/mlb/bottomline/scores';
+  private _MAX_NUM_DISPLAY: number = 3;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const parsedArgs = this._parseArgs(args || []);
+    let teamArg;
+    let sportArg;
+
+    //If no args, team is UCF by default
+    if (parsedArgs === '') {
+      sportArg = 'NCAA';
+      teamArg = 'UCF';
+    } else {
+
+      let argArray = parsedArgs.split(' ');
+      if (argArray.length < 2) {               //Make sure there are 2 arguments given
+        message.reply("Incorrect Amount of Arguments");
+        return;
+      }
+      sportArg = argArray[0];
+
+      argArray = argArray.slice(1);
+      teamArg = argArray.join(' ');
+    }
+
+    let endpoint: string = '';
+    switch (sportArg.toLowerCase()) {
+      case 'ncaa': endpoint = this._NCAA_LINK;
+        break;
+      case 'nfl': endpoint = this._NFL_LINK;
+        break;
+      case 'mlb': endpoint = this._MLB_LINK;
+        break;
+      default: endpoint = this._NCAA_LINK;
+    }
+
+    const visitorDataRegex: RegExp = /[=\^][a-zA-Z]+%20([a-zA-Z]*%20)?[0-9]+/; //Isolates visiting team's data
+    const homeDataRegex: RegExp = /(?<=%20%20)\^?(\(\d+\))?[a-zA-Z(%20)]+%20[0-9]+/; //Isolates home team's data
+
+    try {
+      const response = await this.container.httpService.get(endpoint);
+      const games = response.data.split('?'); //Each game ends with a ?
+
+      const embedBuffer = [];
+      let messagesQueued = 0;
+      let teamFound = false;
+
+      for (const game of games) {
+
+        if (!this._containsAllTokens(game, teamArg.split(' '))) { continue; } //Check if game does not contain all search terms
+        if (messagesQueued >= this._MAX_NUM_DISPLAY) { break; } //Stop if the max amount of messages have been queued
+
+        teamFound = true; //Designates as true if any game was found with matching search term
+
+        let visitorData = game.match(visitorDataRegex);
+        let homeData = game.match(homeDataRegex);
+
+        //Makes sure regex found data
+        if (visitorData == null || homeData == null) { continue; }
+
+        visitorData = visitorData[0];
+        homeData = homeData[0];
+
+        visitorData = visitorData.split('%20');
+        homeData = homeData.split('%20');
+
```

Comment:
nit: only 1 blank line between lines. did you get `prettier` tool?

Reply:
I did, Odd because that double line isnt there in my latest commit. Must've commited without formatting